client/package.json

```diff
@@ -10,6 +10,7 @@
   "license": "ISC",
   "dependencies": {
     "axios": "^0.19.2",
+    "chai-as-promised": "^7.1.1",
     "jquery": "^3.4.1"
```

Comment:
Can we add live-server to the dependencies? Oversight on my part back when I made this.